lib/swarm/tracker/tracker.ex

```diff
@@ -1042,6 +1036,25 @@ defmodule Swarm.Tracker do
     GenStateMachine.cast(from, {:sync_ack, Node.self})
     :keep_state_and_data
   end
+  defp handle_cast({:retry_pending_trackings}, %{pending_trackings: []}) do
+    :keep_state_and_data
+  end
+  defp handle_cast({:retry_pending_trackings}, %{pending_trackings: pending_trackings} = state) do
+    debug "retry pending trackings: #{inspect state.pending_trackings}"
+    state = Enum.reduce(pending_trackings, %TrackerState{state | pending_trackings: []}, fn (tracking, state) ->
```

Comment:
Minor style change here, my preference in new/modified code is to change assignments which break 80 characters into multiple lines and use pipes if applicable to shorten it up; and I also prefer to avoid parens in anonymous function arguments, e.g:

```elixir
state =
  pending_trackings
  |> Enum.reduce(%TrackerState{state | pending_trackings: []}, fn tracking, state ->
    ...
  end)
```